PublishParasoftResults/ParaReportPublishService.ts

```diff
@@ -307,6 +308,12 @@ export class ParaReportPublishService {
 
 
     transformToSarif = (sourcePath: string): void => {
+        let filename = path.basename(sourcePath);
+        if (this.sarifReportMap.has(filename)) {
+            tl.warning(`Due to duplicate name, static analysis report ${sourcePath} get skipped.`);
```

Comment:
Message like: Static analysis report ${sourcePath} is skipped, it has the same name with report sarifReportMap.get(filename) which has already processed.